lib/spack/spack/cmd/checksum.py

```diff
@@ -56,33 +56,34 @@ def checksum(parser, args):
     # Get the package we're going to generate checksums for
     pkg = spack.repo.get(args.package)
 
+    # Otherwise, see what versions we can find online
```

Comment:
I think this comment no longer makes sense here (the online search was moved to later)